### PR TITLE
Updated MBS 0111 adding guidance and removing preview section

### PR DESCRIPTION
**Pull request by jchapman68**
### What is the context of this PR?
The Add Guidance links to forms (https://trello.com/c/csQmUQiQ) has been paired down to just MBS 0111 form with the following changes:-
- Move guidance link to above the Start Survey button
- Add heading above the guidance link to separate it from the legal section
- Normalise headings, h1 in banner, form h2, sections h3

### Dependencies
- https://github.com/ONSdigital/eq-schema-validator/pull/129

### How to review 
From Launch a Survey open each form type to ensure changes to MBS 0111 form and other forms have been successfully applied.

### Checklist

* [ ] MBS 0111 - Guidance link moved to above Start Survey button
* [ ] MBS 0111 - Added heading over Guidance link to separate from Legal section
* [ ] MBS 0111 - Section "Information You Need" has been removed
* [ ] Normalise heading under Form title (h2) as h3s

### CODE REVIEW

**Comment by jchapman68** [comment]
[Guidance links on all live surveys](https://trello.com/c/csQmUQiQ/955-guidance-links-on-all-live-surveys)

File: app/templates/introduction.html
Comment (hamishtaplin) [inline app/templates/introduction.html]: Why add an empty `p` tag?

File: app/helpers/template_helper.py
Comment (samiwel) [inline app/helpers/template_helper.py]: Personally I think this should be named `external_guidance_url` to make it clear what is being passed into the template.

File: data/en/mbs_0111.json
Comment (samiwel) [inline data/en/mbs_0111.json]: Same here regarding renaming.

File: data/en/mbs_0111.json
Comment (samiwel) [inline data/en/mbs_0111.json]: Are we definitely removing this?

**Review by samiwel — CHANGES_REQUESTED** [review]
In general approach looks okay to me. It would be good to get a review from someone on the other team just in case. Also build is failing because the schema does not allow additional properties on the questionnaire. So best to put this on hold and raise a pull request to add `external_guidance_url` as an optional field on eq-schema-validatior repo and get that merged first before merging this one.

File: app/templates/introduction.html
Comment (samiwel) [inline app/templates/introduction.html]: The static content should be wrapped in a `_()` i.e. `_("Guidance to help complete this survey")` because it is subject to being translated. There is also a process to update the pot file with the string to be translated. I think this is documented in the readme IIRC.

**Review by samiwel — CHANGES_REQUESTED** [review]

File: data/en/mbs_0111.json
Comment (jchapman68) [reply data/en/mbs_0111.json]: In conversation the guidance is to replace the preview section so the usage data would have been skewed if both were available to the user.

File: app/templates/introduction.html
Comment (hamishtaplin) [inline app/templates/introduction.html]: Needs more space:

```suggestion
                <p class="u-mb-m"><a rel='noopener noreferrer' target='_blank' href={{ survey_guide }}>Guidance to help complete this survey</a></p>
```

Comment (jchapman68) [reply app/templates/introduction.html]: Done

File: app/templates/introduction.html
Comment (samiwel) [inline app/templates/introduction.html]:
```suggestion
                <p class="u-mb-m">{{_("There is %(opening_tag)s further guidance available%(closing_tag)s with information of what is required for this survey, and help to complete it",opening_tag="<a rel='noopener noreferrer' target='_blank' href="+survey_guidance_url+">",closing_tag="</a>")}}</p>
                {% endif %}
```

Comment (samiwel) [inline app/templates/introduction.html]: Sorry to nitpick, but can you move the `{% endif %}` down to the next line?

**Review by samiwel — CHANGES_REQUESTED** [review]
LGTM, one small comment...

**Review by tgandrews — APPROVED** [review]
LGTM tested locally.

File: app/templates/introduction.html
Comment (jchapman68) [reply app/templates/introduction.html]: Have tidied up the code.

File: app/templates/partials/introduction/basic.html
Comment (pricea1) [inline app/templates/partials/introduction/basic.html:3]: This has changed the font size from 28px to 22px. While it's correct to change to H3 so it is semantically within the H2 form title, there is no requirement to change the design. I think the class name should remain as "u-fs-l".

Also, the h3 content title below ` <h3 class="u-fs-r--b">{{ content.title }}</h3>` now needs to be a h4 so it is semantically within the h3 intro title.

This also applies to preview.html

**Review by pricea1 — CHANGES_REQUESTED** [review]
Just a couple of small changes to ensure the current design isn't changed.

File: app/templates/partials/introduction/basic.html
Comment (jchapman68) [reply app/templates/partials/introduction/basic.html:3]: Have updated basic and preview files as suggested.

Comment (samiwel) [reply app/templates/partials/introduction/basic.html:3]: @hamishtaplin what you reckon?

File: app/templates/partials/introduction/preview.html
Comment (hamishtaplin) [inline app/templates/partials/introduction/preview.html]: Actually, the original change here was correct I think it just wasn't captured in the ticket. The font-size should be `u-fs-m`.

**Review by hamishtaplin — DISMISSED** [review]

File: app/templates/partials/introduction/basic.html
Comment (hamishtaplin) [reply app/templates/partials/introduction/basic.html:3]: Yeah Andrew was correct about the heading structure.

File: app/templates/partials/introduction/preview.html
Comment (jchapman68) [reply app/templates/partials/introduction/preview.html]: Ok - will revert back for preview, do you want the similar change in basic.html also reverted?

File: app/templates/introduction.html
Comment (jchapman68) [reply app/templates/introduction.html]: Can't see that tag in the file!

File: app/templates/introduction.html
Comment (jchapman68) [reply app/templates/introduction.html]: Not sure if this is still a problem

File: app/templates/partials/introduction/preview.html
Comment (hamishtaplin) [reply app/templates/partials/introduction/preview.html]: No that's fine

**Review by samiwel — APPROVED** [review]

**Comment by samiwel** [comment]
@hamishtaplin @pricea1 Please can you take another look to make sure you're happy with the changes? You still have blocking reviews.

**Review by tgandrews — APPROVED** [review]
LGTM if we are happy taking the small template changes

**Review by pricea1 — APPROVED** [review]
LGTM

**Comment by samiwel** [comment]
Hey @jchapman68 can you rebase and squash the commits down then we'll get this merged?

**Pull request merged**
